### ISSUE TITLE
Please add the bootloder files

### ISSUE THREAD

**Issue by andres-asm** [clockworkpi/bootloader#1]
I want to make a lakka image for the GS

**Comment by letarg0** [comment]
yes! add files

**Comment by laosb** [comment]
Please.

**Comment by duckyvirus** [comment]
since it doesnt seem that they intend to share them yet, I will look into extracting them from the uImage file itself and see if i can upload. if i cant I will link them here

**Comment by kmmndr** [comment]
The Gameshell seemed so perfect at first glance, but the bootloader is still missing.
Not much more information on the [forum](https://forum.clockworkpi.com/t/cpi-kernel-source/2133/2) apart from the fact that it seems to be locked by Allwinner.
Any news since last year ? @allwinner-zh ?

**Comment by arthurlutz** [comment]
@kmmndr I agree with you, the bootloader is not the only part that is missing in my opinion, I'm not getting any answers on how the OS images are being generated : https://forum.clockworkpi.com/t/how-are-the-gameshell-os-image-files-generated/4474

**Comment by AcrylicShrimp** [comment]
Still no active?
